app/forms.py

```diff
@@ -73,6 +73,17 @@ class Meta:  #pylint: disable=no-init,missing-docstring,old-style-class,too-few-
         only = ['position', 'organization', 'organization_type', 'city',
                 'country']
 
+    @classmethod
+    def is_not_empty(cls, user):
+        '''
+        Returns whether or not the user has filled out any of the
+        fields in the form. Used to determine whether to skip past
+        this form when resuming the registration process.
+        '''
+
+        return (user.position or user.organization or user.city or
+                user.country)
```

Comment:
There didn't seem to be a particularly reliable way of doing this by introspecting the form itself, so I just manually coded it, and put it in the form itself so that hopefully it'll get updated whenever the form fields are.